packages/task/test-resources/.theia/tasks.json

```diff
@@ -55,6 +55,57 @@
                     "dir"
                 ]
             }
+        },
+        {
+          "label": "Test compile problemMatcher",
+          "type": "shell",
+          "command": "tsc ${file}",
+          "problemMatcher": [
+              "$tsc"
+          ]
+        },
+        {
+          "label": "Test presentation",
+          "type": "shell",
+          "command": "echo New window opened!",
+          "presentation": {
+              "reveal": "always",
+              "panel": "new",
+              "clear": true
+          }
+        },
+        {
+          "label": "Test Compound tasks",
+            "type": "shell",
+            "command": "echo Test Compound is done!",
+            "dependsOn": [
+                "test task",
+                "test input variables"
+            ],
+            "presentation": {
+                "reveal": "always",
+                "panel": "new",
+                "clear": true
+            }
```

Comment:
Is there a reason we want to repeat the same `presentation` options for:
- `test presentation`
- `test compound task`
- `test input variables`

All three of them use:

```json
"presentation": {
  "reveal": "always",
  "panel": "new",
  "clear": true
}
```